.devcontainer/devcontainer.json

```diff
@@ -24,7 +24,7 @@
     "ghcr.io/devcontainers/features/node": "lts",
     "ghcr.io/devcontainers/features/go": "latest",
     "ghcr.io/devcontainers/features/ruby": "3.3.6",
-    "ghcr.io/devcontainers/features/rust": "latest",
+    "ghcr.io/devcontainers/features/rust": "1.86.0",
```

Comment:
This ended up being necessary to get my devcontainer working, Probably this is a point in time issue (I'm told there was just a new version) that can be reverted before I merge.